Makefile

```diff
@@ -3,6 +3,7 @@ PREFIX ?= /usr/local
 install:
 	bin/check
 	docker pull codeclimate/codeclimate:latest
+	docker images | grep codeclimate/codeclimate- | awk '{print $1}' | while read image; do docker pull $image; done
```

Comment:
Generally `grep <re> | awk { <cmds> }` can be replaced with `awk /<re>/ { <cmds> }`

```
... | ' awk /codeclimate\/codeclimate-/ { print $1 }' ...
```